src/browser/modules/EditorInfo/EditorInfo.jsx

```diff
@@ -14,6 +14,7 @@ export class EditorInfo extends Component {
     return (
       <div>
         <DisplayNodeDetails
+          label={this.props.selectedItem._fields[0].labels}
```

Comment:
and if we are passing labels then term the prop name as labels. 
this is to avoid ambiguity.